scripts/make_width_table.py

```diff
@@ -0,0 +1,50 @@
+import sys
```

Comment:
Please add a docstring explaining what this file does and when it should be run.